README.md

```diff
@@ -138,63 +126,163 @@ In support of the various browser privacy models, first-party sets only control
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+<strong>Subset type</strong></th>
+      <th><br>
+<strong>Subset definition</strong></th>
+      <th><br>
+<strong>Example</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><br>
+ccTLD (country code Top Level Domain)</td>
+      <td><br>
+Reserved for variations for a particular country or a geographical area. <br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA.co.uk<br>
+<br>
+exampleA.ca</td>
+    </tr>
+    <tr>
+      <td><br>
+common eTLD (effective Top Level Domain)</td>
+      <td><br>
+Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA.gov.uk<br>
+<br>
+exampleB.gov.uk</td>
+    </tr>
+    <tr>
+      <td><br>
+service</td>
+      <td><br>
+Reserved for utility or sandbox domains.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA-usercontent.com<br>
+<br>
+exampleA-cdn.net</td>
+    </tr>
+    <tr>
+      <td><br>
+associated</td>
+      <td><br>
+Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td><br>
+exampleA-affiliated.com<br>
+<br>
+exampleB.com*<br>
+<br>
+exampleC.com*<br>
+<br>
+<br>
+*where exampleB and exampleC are separately owned websites, but clearly present their affiliation with exampleA to users</td>
+    </tr>
+  </tbody>
+</table>
+
+While we think this subset framework has the clear benefit of furthering transparency around why a domain has been added to a set, the primary value to this framework is that the browser could handle each subset differently:
+
+<table>
+  <thead>
+    <tr>
+      <th><strong>Subset type</strong></th>
+      <th><strong>Subset definition</strong></th>
+      <th><strong>Example browser handling policy</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>ccTLD (country code Top Level Domain)</td>
+      <td>Reserved for variations for a particular country or a geographical area. <br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>common eTLD (effective Top Level Domain)</td>
+      <td>Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>service</td>
+      <td>Reserved for utility or sandbox domains.<br>
```

Comment:
Is a utility or sandbox domain always a domain that is not intended to appear in search results? Another possible rule here would be to require that service domains block general-interest web search engines in their robots.txt.

Reply:
Yes, I think that this is a great approach to service/utility/sandbox domains, where we can make a few strong assumptions along the lines of "this is not a self-sufficient site that users would discover and interact with, except when integrating with other sites in the set", then build hard technical constraints that match these, like requiring of blocking crawling.

This is something we should work with developers to figure out, to understand potential use cases of "service" domains and which constraints would break valid use cases. I think this is partially tracked in #95 and we might have time to discuss it on the WICG call.